#include "std_testcase.h"

#include <string>
#include <iostream>
#include <stdexcept>

using namespace std; 

namespace var1
{



void func1()
{
    int j;
    for(j = 0; j < 1; j++)
    {
        throw exception();
    }
}

} 



using namespace var1; 

int main(int argc, char * argv[])
{
    srand( (unsigned)time(NULL) );

    printLine("Calling ...");
    func1();
    printLine("Finished");
    return 0;
}

